### ISSUE TITLE
"FORBIDDEN" error modal dialog in app when edit a (M2A) related item

### ISSUE THREAD

**Issue by cschuller** [directus/directus#17406]
### Describe the Bug

The Directus app shows a "Forbidden" error modal dialog box for an M2A item.
An API call is used to GET the relation with a wrong ID. (IDs between relation and related items are mixed up)

#### Setup
- Create new collection (Name: "testA", PKF: "id", Type: "**Generated UIDD**") 
- Create new collection (Name: "testB", PKF: "id", Type: "**Generated UIDD**") 
- Create new collection (Name: "testC", PKF: "id", Type: "**Generated UIDD**") 

- Edit collection "testA"
- Add field "childs" (M2A) with "testB" and "testC"

#### Content
- Create new "testA" content
- Create new "childs" entry with "testB"
- Save

#### Error Steps
- Open created "testA" content
- Click on item in "childs" list
- "Forbidden" error is displayed

#### UI
The Directus app shows a "Forbidden" error modal dialog box, because a wrong API request was sent.

#### Network TAB
```
Request URL: https://example.com/items/testA_childs/0caeae26-eec9-443b-89a8-3b30aaf092cf
Request Method: GET
Status Code: 403 
```

## LOG
```
[23:18:40] GET /items/testA_childs/0caeae26-eec9-443b-89a8-3b30aaf092cf 403 18ms
```

### What version of Directus are you using?

9.22.4

### What version of Node.js are you using?

18.13.0

### What database are you using?

SQLite 5.1.4

### What browser are you using?

Edge, Chrome

### How are you deploying Directus?

locally

## 🌵 Analysis
When opening the "childs" item in the right side panel/drawer a wrong API request is sent.
The reference collection "testA_childs" is using numbers as IDs.
The "childs" items (testB or testC) are using UUIDs, so the request 
`GET /items/testA_childs/0caeae26-eec9-443b-89a8-3b30aaf092cf` cannot be correct?

**Comment by paescuj** [comment]
Thanks for opening this issue! However this is a duplicate of #17108, so I'll proceed to close this but if you think this is actually a different issue, then please do not hesitate to comment.

This issue has been fixed by #17210 which will be available in the next release.

**Comment by cschuller** [comment]
@paescuj Sorry, I somehow did not find the related issue with my search. 
Anyway a coffee was sponsored ;-)
